clang/include/clang/Basic/Attr.td

```diff
@@ -1121,6 +1121,14 @@ def CoroLifetimeBound : InheritableAttr {
   let SimpleHandler = 1;
 }
 
+def CoroNotLifetimeBound : InheritableAttr {
```

Comment:
Any alternatives to this name? Maybe `coro_disable_lifetime_bound`?

The current name confused me a little because `coro_lifetime_bound` is supposed to be applied to types so I naturally expect `coro_not_lifetime_bound` to be applied to types too. In practice, it's applied to functions so having a different name would help me avoid confusion.

WDYT?

Reply:
SG.